ai_genomics/analysis/tag_evaluation/add_oa_tag_thresholds.py

```diff
@@ -0,0 +1,73 @@
+"""Script to add OA tag thresholds in labelled tag extraction validation dataset."""
+import ast
+import pandas as pd
+
+from ai_genomics.getters.data_getters import load_s3_data, save_to_s3
+from ai_genomics import bucket_name
+from ai_genomics.analysis.tag_evaluation.make_tag_validation_dataset import (
+    load_sample_tags,
+)
+
+LABELLED_TAG_EVALUATION_DATASET = load_s3_data(
+    bucket_name,
+    "inputs/ai_genomics_samples/labelled_ai_genomics_sample_validation_set.csv",
+)
+
+
+def get_oa_tags_above_threshold(
```

Comment:
Just fyi another way of performing this would be to chain [`pandas.explode`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.explode.html?highlight=explode#pandas.DataFrame.explode), apply the threshold and then use a `groupby.apply(list)` to recombine.

Although now I've typed that out, I think it could be abominably slow and my suggestion should really be to write it in vanilla Python, but this will fly 😄